encryption/encryption.cc

```diff
@@ -15,6 +15,17 @@ namespace rocksdb {
 namespace encryption {
 
 namespace {
+// For some files, we don't require them to be encrypted.
```

Comment:
New changes seem unrelated to this PR?

Reply:
@tabokie We find that rocksdb rely on `rename` being atomic to update the `CURRENT` file when creating a new manifest. For a quick fix we are to exclude the `CURRENT` file from being encrypted.